changelog.d/20240730_020618_derek_refactor_login_flow_manager_interactions.rst

```diff
@@ -0,0 +1,29 @@
+
+Changed
+~~~~~~~
+
+-   Changed the experimental ``GlobusApp`` class in the following ways (:pr:`NUMBER`):
+
+    -   ``LoginFlowManagers`` now insert ``GlobusApp.app_name`` into any Native
```

Comment:
I think we should either always capitalize "Native Client" or always lowercase (except at beginning of a sentence). I don't think mixed case like this makes sense (but if you disagree, perhaps I can be persuaded!)

Suggesting the latter, as I believe it's consistent with existing usage elsewhere.

```suggestion
    -   ``LoginFlowManagers`` now insert ``GlobusApp.app_name`` into any native
```